src/config/ConfigManager.cpp

```diff
@@ -201,3 +201,14 @@ void CConfigManager::handleUnloadAll(const std::string& COMMAND, const std::stri
     for (auto& tu : toUnload)
         g_pHyprpaper->unloadWallpaper(tu);
 }
+
```

Comment:
why are there 4 newlines here? only 1 plox

Reply:
Yeah sorry about that, I fixed those lines and the returned substring's length argument.